mono/mini/method-to-ir.c

```diff
@@ -9024,8 +9033,13 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 			if (ins)
 				goto call_end;
 
+#if 0 // debug code -- attempt to tailcall every call followed by a ret
```

Comment:
Use a `#define` which conveys what you are trying to debug. Also you can factor the `gboolean inst_tailcall` to reduce chances of code bitrot.